mlir/test/mlir-miopen-driver/populate_host_convert.mlir

```diff
@@ -1,6 +1,6 @@
 // RUN: mlir-miopen-driver -p -ph -pr | FileCheck %s --check-prefix=F32
 // RUN: mlir-miopen-driver -p -ph -pr -t f16 | FileCheck %s --check-prefix=F16
-// RUN: mlir-miopen-driver -p -ph -pr -t bf16 | FileCheck %s --check-prefix=BF16
+// FIXME: BF16 use another function, so skip it
```

Comment:
remove FIXME.

Reply:
and re-enable BF16 RUN command.

Reply:
but I don't use convert function when bf16

Reply:
test added